css-canvas-medical/styles.css

```diff
@@ -0,0 +1,197 @@
+#privacy {
+  margin-right: 40px;
+}
+
+#end {
+  padding: 100px 10%;
+}
+
+.indented {
+  padding-left: 10%;
+  padding-right: 10%;
+}
+
+.relative {
+  position: relative;
+}
+
+.absolute {
+  position: absolute;
+}
+
+.border-bottom {
+  border-bottom: 1px solid rgb(231, 233, 236);
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+h2 {
+  font-family: 'Tiempos Headline', serif;
+}
+
+h3 {
+  font-family: 'Tiempos Headline', serif;
+  font-weight: 300;
+  margin: 0;
+}
+
+.blue-button {
+  background-color: hsl(213deg, 42%, 28%);
+  padding: 20px 50px;
+  border-radius: 4px;
+}
+
+.blue-button:hover {
+  background-color: hsl(213deg, 42%, 18%);
+  text-decoration: none;
+}
+
+.s-three .blue-button {
+  display: block;
+}
+
```

Comment:
Why does this need to be `display: block`? You have this wrapped in a `div` which is a block element as well as it being in a `flex` container so this is moving along a row.

Reply:
If it's not block, it jumps ~20px above the div that contains it. I'd better take a thorough look through my CSS to find out why.